model/cloud.go

```diff
@@ -312,11 +312,16 @@ type CreateSubscriptionRequest struct {
 	DiscountID            string   `json:"discount_id"`
 }
 
-type DowngradeFeedback struct {
+type Feedback struct {
 	Reason   string `json:"reason"`
 	Comments string `json:"comments"`
 }
 
+type WorkspaceDeletionRequest struct {
+	SubscriptionID string `json:"subscription_id"`
+	Feedback
```

Comment:
Not a showstopper but by embedding the struct, it makes it available on the parent, so someone could confusingly do

```go
deletionRequest := WorkspaceDeletionRequest{
  SubscriptionID: "sub_something",
  Feedback: Feedback{
    Reason: "some reason",
    Comments: "some comments",
  }
}

deletionRequest.ToMap()
```
Calling ToMap on the request would only return the map of the comments, not of the structure as a whole. Again, not likely to happen, but possibly could be confusing in the future. Could do like SubscriptionChange does and make it a member instead of embedded.